bot/cogs/bot.py

```diff
@@ -15,6 +18,8 @@ class Bot:
 
     def __init__(self, bot: AutoShardedBot):
         self.bot = bot
+        self.code_block_channels = {303906576991780866: 0, 303906556754395136: 0, 303906514266226689: 0, 267624335836053506: 0}
+        # Stores allowed channels plus unix timestamp from last call
```

Comment:
This comment is still in the wrong place. Comments should not be placed after the line they document.